harvard-lts/hdc-integration-tests
# Check S3 and Dropbox for Exports

**Check S3 and Dropbox for Exports**
* * *

**JIRA Ticket**: (https://app.zenhub.com/workspaces/hdc-62163f60fb08c60011fa4795/issues/harvard-lts/hdc/199)

# What does this Pull Request do?
Checks the dropbox and s3 bucket for exports from Dataverse

# How should this be tested?

Deployed to Dev, run:
`curl -k https://<cloud-dev-1>:10582/DIMS/DVIngest`
You should see `"num_failed": 0`

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n/a
- integration tests? n/a

# Interested parties
@ives1227 @dl-maura
